src/odm_sharing/private/utils.py

```diff
@@ -0,0 +1,6 @@
+def qt(x: str) -> str:
+    '''quote `x`'''
+    return f"'{x}'"
+
+def not_empty(x) -> bool:  # type: ignore
```

Comment:
Why do we have the `type: ignore`?

Reply:
since x can be Any, and Any isn't accepted by mypy, so it's better to ignore it all together.

Reply:
That makes sense. Can you add a comment about it?